docs/posts/lms-for-data/index.qmd

```diff
@@ -0,0 +1,690 @@
+---
+title: "Using language models for data"
```

Comment:
`for data` is a bit generic. maybe `for data synthesis and analytics`?

Reply:
are there use cases you have in mind that aren't covered in this post?